README.md

```diff
@@ -902,15 +902,21 @@ If you also want to stash untracked files, use `-u` option.
 $ git stash -u
 ```
 
-<a name="stash-specific-file"></a>
-### Stash specific file
+<a name="stash-specific-files"></a>
+### Stash specific files
 
 To stash only one file from your working directory
 
 ```sh
 $ git stash push working-directory-path/filename.ext
 ```
 
+To stash multiple files from your working directory
+
+```sh
+$ git stash push working-directory-path/filename1.ext <space> working-directory-path/filename2.ext
```

Comment:
What does the `<space>` signify?

Reply:
I don't think that's useful - a space itself would do this. Can we remove it?

Reply:
something like this:  

$ git stash push working-directory-path/filename1.ext working-directory-path/filename2.ext ?

Reply:
Yep!

Reply:
Fix it